style2.css

```diff
@@ -0,0 +1,529 @@
+@media screen and (max-width: 1680px) {
+  body {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+@media screen and (max-width: 1440px) {
+  .main__pic {
+    margin: -100px 0 0 0;
+  }
+}
+
+@media screen and (max-width: 1120px) {
+  .header__menu-link {
+    font-size: 14px;
+  }
+
+  .main__pic {
+    margin: -80px 0 0 0;
+  }
+}
+
+@media screen and (max-width: 1090px) {
+  .header__menu-link {
+    font-size: 13px;
+  }
+
+  .main__pic {
+    margin: -70px 0 0 0;
+  }
+
+  .sec__container {
+   display: flex;
+   flex-direction: column;
+   
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)